### ISSUE TITLE
VFlow does not work with maxHeight

### ISSUE THREAD

**Issue by MattNewberry** [tevelee/SwiftUI-Flow#11]
The readme states to set the `maxHeight` to `infinity`, however I'm only able to get `VFlow` to work correctly after specifying an exact height value. 

```
VFlow {
    ForEach(colors, id: \.description) { color in
        RoundedRectangle(cornerRadius: 10)
            .fill(color.gradient)
            .frame(width: 50, height: Double.random(in: 40...60))
    }
}
.frame(maxHeight: 300)
```

However, this produces: 
<img width="383" alt="Screenshot 2024-04-23 at 12 58 32 PM" src="https://github.com/tevelee/SwiftUI-Flow/assets/108004/0eb3805b-254c-4bee-abd2-d3991b02fb86">

If I change this to a manually specified height, I'm able to correctly see the expected result:
```
VFlow {
    ForEach(colors, id: \.description) { color in
        RoundedRectangle(cornerRadius: 10)
            .fill(color.gradient)
            .frame(width: 50, height: Double.random(in: 40...60))
    }
}
.frame(height: 300)
```
<img width="127" alt="Screenshot 2024-04-23 at 12 58 11 PM" src="https://github.com/tevelee/SwiftUI-Flow/assets/108004/99d27a8a-4300-43ed-83d1-686a3f412d89">

The problem here, though, is knowing the height ahead of time to specify.

**Comment by tevelee** [comment]
> The readme states to set the maxHeight to infinity

Where does README.md state this?

**Comment by MattNewberry** [comment]
My mistake! Let me rephrase - without setting a `maxHeight`, `VFlow` does not work correctly. Am I thinking of this incorrectly? I'm picturing this taking up all the height it needs but being constrained by `width`, instead.
